src/backend/devices/kpro/constants.py

```diff
@@ -6,6 +6,11 @@
 KPRO23_ID = 23  # v2 and v3 works equally so I mixed both version numbers
 KPRO4_ID = 4
 
+# s300 device ids
```

Comment:
this constants should be in s300/constants.py and then imported wherever is necessary